fellow-submissions/stephanie-smith.md

```diff
@@ -0,0 +1,108 @@
+# Problem Set 2.4 - Scope
+
+**1. How do variables declared with `let`, `const`, and `var` differ? Be sure to touch on _scope_, _reassignment_, and _hoisting_ for each.**
+- Variables declared with `var` can be reassigned as often as you want, and you can access them regardless of scope. However, if you are going to use `var` with function expressions, you cannot hoist them, so you'll have to declare it after the function expression is initialized.
```

Comment:
There are some key misunderstandings here:
* `var` cannot be accessed regardless of scope. All variables have a scope and cannot be accessed outside of that scope. However, the difference is in how the scope is defined. `var` scopes are defined by functions. `let`/`const` are block scoped.
* Also, variables are hoisted just like function declarations are but their initial values differ depending on how they were declared. Be sure that you have read [this article](https://blog.bitsrc.io/hoisting-in-modern-javascript-let-const-and-var-b290405adfda) from the lesson.

You were correct about reassignment with `let`/`var` vs `const`.